ada_feeding/ada_feeding/helpers.py

```diff
@@ -331,12 +331,15 @@ def get_moveit2_object(
     # the same object.
     moveit2_blackboard_key = "/moveit2"
     moveit2_lock_blackboard_key = "/moveit2_lock"
+    end_effector_tool_blackboard_key = "/end_effector_tool"
```

Comment:
In the earlier version of this function, if the BB keys don't exist, it wrote them. That is no longer the case for the EE tool key. Two requested changes:

1. Consider having some default behavior where if the key isn't set, it writes `none` to the BB?
2. Irrespective of the above, add to the function docstring the expectation that the EE tool key is set, and what happens if/when it is not.